src/canvas/DesktopElement.ts

```diff
@@ -402,7 +404,10 @@ export default class GmfDesktopCanvas extends BaseElement {
       panels.getActiveToolPanel().subscribe({
         next: (panel: string) => {
           const styles = getComputedStyle(document.documentElement);
-          let width = styles.getPropertyValue(`--right-panel-width-${panel}`);
+          let width = this.toolsPanelWidth_[panel];
+          if (!width) {
```

Comment:
There is the same if two line below, but not for the same thing, please check

Reply:
It's for the default values :-)

Reply:
Comments added :-)